src/it/resources/com/sun/checkstyle/test/chapter4indentation/rule41linelength/InputLineLength.java

```diff
@@ -0,0 +1,90 @@
+package com.sun.checkstyle.test.chapter4indentation.rule41linelength;
+
+public class InputLineLength
```

Comment:
Please make test input file more simple, this seems more confusing than it needs to be. It would be good to show violations on both comments and code.